Src/Support/Google.Apis.Auth.Mvc/Google.Apis.Auth.Mvc.csproj

```diff
@@ -43,6 +43,7 @@ Supported Platforms:
 
     <PackageReference Include="ConfigureAwaitChecker.Analyzer" Version="5.0.0" PrivateAssets="All" />
     <PackageReference Include="SourceLink.Create.CommandLine" Version="2.8.3" PrivateAssets="All" />
+	<PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies.net45" Version="1.0.2" PrivateAssets="All" />
```

Comment:
Nit: change tabs to spaces.

Reply:
Possibly always target net45 (so remove lines 16 and 17 and unconditionalize line 15)?